dashboard/django/stats/templates/stats/clusters.html

```diff
@@ -0,0 +1,111 @@
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+{% extends "stats/base.html" %}
+
+{% load humanize %}
+{% load table %}
+{% load stats_extras %}
+
+{% block title %}Clusters{% endblock %}
+
+{% block content %}
+
+
+<table>
+<thead>
+    <tr>
+        {% column_header clusters 'name' 'Cluster' %}
+        {% column_header clusters 'numTopics' 'Topics' %}
+
+        {% column_header clusters 'localRateIn' 'Local Rate In' %}
+        {% column_header clusters 'localRateOut' 'Local Rate Out' %}
+        {% column_header clusters 'replicationRateIn' 'Replication Rate In' %}
+        {% column_header clusters 'replicationRateOut' 'Replication Rate Out' %}
+
+        {% column_header clusters 'localBacklog' 'Local Backlog' %}
+        {% column_header clusters 'replicationBacklog' 'Replication Backlog' %}
+        {% column_header clusters 'storage' 'Storage' %}
+    </tr>
+</thead>
+<tbody>
+
+{% for cluster in clusters.results %}
+    <tr class="{% cycle 'row1' 'row2' %}">
+        <th>
+            <a href="#">{{cluster.name}}</a>
+        </th>
+
+        <td>{{cluster.numTopics | intcomma}}</td>
+        <td>{{cluster.localRateIn | intcomma}}</td>
+        <td>{{cluster.localRateOut | intcomma}}</td>
+        <td>{{cluster.replicationRateIn | intcomma}}</td>
+        <td>{{cluster.replicationRateOut | intcomma}}</td>
+
+        <td>{{cluster.localBacklog | intcomma}}</td>
+        <td>{{cluster.replicationBacklog | intcomma}}</td>
+        <td>{{cluster.storage | filesizeformat}}</td>
+    </tr>
+
+    <tr class="{% cycle 'row1' 'row2' %}">
+        <td></td>
+        <td colspan="6">
+            <table>
+            <thead>
+                <tr>
+                    <th>Remote</th>
+                    <th>Address</th>
+                    <th title="Msg/s">Rate out</th>
+                    <th title="bytes/s">Throughput out</th>
+                    <th title="Msg/s">Redelivery rate</th>
+                    <th>Connected since</th>
+                    <th>Available permits</th>
+                    <th title="Messages">Unacked</th>
+                    <th title="Blocked for max unacked messages">Blocked</th>
+                </tr>
+            </thead>
+            <tbody>
+            {% for consumerStats in consumers %}
+                <tr>
+                    <td>{{consumerStats.consumerName}}</td>
+                    <td>{{consumerStats.address}}</td>
+                    <td>{{consumerStats.msgRateOut | intcomma}}</td>
+                    <td>{{consumerStats.msgThroughputOut | intcomma}}</td>
+                    <td>{{consumerStats.msgRateRedeliver | intcomma}}</td>
+                    <td title="{{consumerStats.connectedSince}}">{{consumerStats.connectedSince | naturaltime}}</td>
```

Comment:
should we give timestamp instead of `naturaltime` incase if consumer is connected since 20 days then it will be easy for user to understand the exact date rather `20 days ago`.